individual.cpp

```diff
@@ -108,5 +108,25 @@ void test_individual()
     individual i{i_p};
     assert(i.get_net() == network{net_par});
   }
+
+  #define FIX_ISSUE_125
+  #ifdef FIX_ISSUE_125
+  ///individual stores a pointer to a base network
+    {
+
+      net_param net_par;
+      network n{net_par};
+
+      std::shared_ptr<network> n_ptr(new network(net_par));
+
+      ind_param i_p{net_par};
+      individual i{i_p};
+
+
+      assert(i.get_net() == *n_ptr);
+      assert(!(i.get_pointer_to_net() == n_ptr));
```

Comment:
I actually have created a function like this has well and named it `get_net_ptr()`.
I prefer my naming, whihc one do you prefer?